libsel4vm/src/arch/arm/vgic/vgic.c

```diff
@@ -591,6 +589,19 @@ static int vgic_dist_disable_irq(struct vgic_dist_device *d, vm_vcpu_t *vcpu, in
     return 0;
 }
 
+static int vgic_irq_queued(vgic_t *vgic, vm_vcpu_t *vcpu, int irq)
```

Comment:
nitpicking: There is `bool`, so this could start using it:
```suggestion
static bool vgic_is_irq_queued(vgic_t *vgic, vm_vcpu_t *vcpu, int irq)
```
and then return `true` or `false`

Reply:
Not nitpicking at all IMO, but I realized there's is_pending() that we can use. Remove this addition altogether.